apps/meteor/tests/data/permissions.helper.ts

```diff
@@ -63,3 +64,30 @@ export const removePermissions = async (perms: string[]) => {
 export const addPermissions = async (perms: { [key: string]: string[] }) => {
 	await updateManyPermissions(perms);
 };
+
+type Permission = typeof permissions[number]['_id']
+
+export const removePermissionFromAllRoles = async (permission: Permission) => {
+    await updatePermission(permission, []);
+};
+
+export const addPermissionToDefaultRoles = async (permission: Permission) => {
```

Comment:
I don't know how the name of this function makes sense. after a little effort I understood that this function RESTORES the default values (but with a secret even if it is not EE), does it make sense to have this condition inside the test?

maybe there should be an endpoint that does that, don't you think?